File: Algoritmos/Algoritmos.Test/Sorting/InsertionSortTest.cs
using Algoritmos.Sorting;

namespace Algoritmos.Test.Sorting;

public class InsertionSortTest
{
    [Fact]
    public void Sort()
    {
        int[] inputs = { 12, 11, 13, 5, 6 };

        var result = InsertionSort.Sort(inputs);
        
        Assert.Equal(5, result[0]);
        Assert.Equal(6, result[1]);
        Assert.Equal(11, result[2]);
        Assert.Equal(12, result[3]);
        Assert.Equal(13, result[4]);
    }
}

File: typescript/src/sorting/insertion-sort.test.ts
import {algorithmForInsertionSort} from "./insertion-sort";

test('insertionSort', () => {
    const numbers = [5, 2, 9, 1, 5, 6];
    const result = algorithmForInsertionSort(numbers);

    expect(result).toStrictEqual([1, 2, 5, 5, 6, 9]);
})